src/Block/Service/AbstractClassificationBlockService.php

```diff
@@ -66,7 +71,7 @@ final protected function getFormAdminType(FormMapper $formMapper, AdminInterface
             'required' => false,
         ], $fieldOptions);
 
-        return $formMapper->create($formField, ModelListType::class, $fieldOptions);
+        return $formMapper->getFormBuilder()->create($formField, ModelListType::class, $fieldOptions);
```

Comment:
Here I am trying to avoid the deprecated call to ->create(). Following same strategy as we did for blocks on PageBundle: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Block/SharedBlockBlockService.php#L148

There is not an easy way to test if the whole block works or not, but I think I will leave it for a follow up PR. My guess is that those blocks does not work, with or without this PR.